### PR TITLE
Fix eslint rule: no-useless-return

### PR DESCRIPTION
**Pull request by aljordan82**
## What's this PR do?
Fix eslint rule: no-useless-return

### CODE REVIEW

File: gulp/.eslintrc.json [hunk 1]
```diff
@@ -9,19 +9,14 @@
...
     "object-shorthand": 0,
     "comma-dangle": [1, "always-multiline"],
     "function-paren-newline": 0,
-    //  "block-spacing": 0,
-    //  "array-bracket-spacing": 0,
     "no-continue": 0, // this can stay off
-    //  "no-confusing-arrow": 0,
     "no-multi-assign": 0,
     "no-underscore-dangle": 0,
-    "no-useless-return": 0,
```
Comment (darrint) [inline gulp/.eslintrc.json]: Why are we removing the rule if we're fixing it?

Comment (aljordan82) [reply gulp/.eslintrc.json]: Rules are enabled by default.

**Comment by darrint** [comment]
I tried to test the changed code but was blocked by this bug: https://directemployersfoundation.atlassian.net/browse/PD-4023

**Comment by darrint** [comment]
Hm. I think I pressed the merge button by mistake. I'm going to leave it since this is such a low-risk PR. The above bug is noted though.

**Pull request merged**